src/sphinxtheme/plone/plone_basic/static/css/ploneorg5.css

```
@@ -98,3 +98,17 @@ div.admonition-description {
   border-color: #dedede;
   color: #086ca3;
 }
+
```

I missed this earlier. The link needs a `hover` style.

```suggestion
a#floating-discord-btn:hover {
  background: #006b56;
}
```